source/features/follow-file-renames.tsx

```diff
@@ -15,7 +16,7 @@ interface File {
 async function findRename(lastCommitOnPage: string): Promise<File[]> {
 	// API v4 doesn't support it: https://github.community/t/what-is-the-corresponding-object-in-graphql-api-v4-for-patch-which-is-available-in-rest-api-v3/13590
 	const {files} = await api.v3(`commits/${lastCommitOnPage}`);
-	return files;
+	return files as File[];
```

Comment:
Redundant

Reply:
Takes away a xo warning

Reply:
I think we'll have to disable those rules altogether, I didn't notice we have 170 warnings and they're all around API calls (which we won't type)
```suggestion
	return files;
```